features/support/env.rb

```diff
@@ -81,3 +86,53 @@ def output_is_script_readable(raw_stdout)
 Then(/^stdout from "([^"]*)" should match \/(.*)\/$/) do |cmd, regexp|
   aruba.command_monitor.find(Aruba.platform.detect_ruby(cmd)).send(:stdout) =~ /#{regexp}/
 end
+
+# track service process ID
+@service_current_process_id = -1
+
+# Note: Only for services supported through systemctl. Not for 'kubernetes'.
+Then(/^the service "([^"]*)" should be ([^"]*)$/) do |service, operation|
+  run("vagrant ssh -c \"sudo systemctl status #{service}\"")
+
+  if ['running', 'restarted'].include? operation
+    exit_code = 0
+    regexp = /Active: active \(running\)/
+  elsif operation == 'stopped'
+    exit_code = 3
```

Comment:
These exit codes are result of `systemctl` command and it is different than our VSM exit codes.